cli/src/index.ts

```diff
@@ -16,9 +19,8 @@ program
     .addOption(new Option('-i, --instantiation <file>', 'Path to an instantiation of a CALM pattern.').conflicts('pattern'))
     .addOption(new Option('-p, --pattern <file>', 'Path to a CALM pattern.').conflicts('instantiation'))
     .requiredOption('-o, --output <file>', 'Path location at which to output the SVG.', 'calm-visualization.svg')
-    .option('-v, --verbose', 'Enable verbose logging.', false)
-    .action(async (options) => { 
-        if (!options.instantiation && ! options.pattern) {
+    .option('-v, --verbose', 'Enable verbose logging.', false)    .action(async (options) => {
```

Comment:
looks like a new line was accidentally deleted? `.action(...` should be on a new line

Reply:
Yeah not going to pretend I understand how this happened - it's resolved now.